samples/boards/npcx7m6fb_evb/demo/src/main.c

```diff
@@ -0,0 +1,211 @@
+/*
+ * Copyright (c) 2020 Nuvoton Technology Corporation.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr.h>
+#include <device.h>
+#include <devicetree.h>
+#include <drivers/gpio.h>
+#include <drivers/uart.h>
+#include <shell/shell.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+#include <linker/linker-defs.h>
+
+void main(void)
+{
+	const struct device *uart_dev;
+	int ret;
+
+
+	/* Get uart_dev device */
+	uart_dev = device_get_binding("UART_1");
+	if (uart_dev == NULL) {
+		printk("Error: didn't find %x device\n", uart_dev);
+		return;
+	}
+	printk("Start the shell test!!!");
```

Comment:
It is risky because this can result in concurrent UART access between the shell and the `printk` function. If you are using the Logger module in the project please add the following configuration in the prj.conf file:
`CONFIG_LOG_PRINTK=y`.
The shell will take care to print this printk message.

If you are not using the Logger but you are using a UART shell than you can print this message using the shell.

add:
`#include <shell/shell_uart>`

Replace the `printk` with the following code:

```
const struct shell *shell = shell_backend_uart_get_ptr();
shell_print(shell, "Start the shell test!!!");
```